cmd/commands/mons.go

```diff
@@ -39,12 +39,16 @@ var MonCmd = &cobra.Command{
 // RestoreQuorum represents the mons command
 var RestoreQuorum = &cobra.Command{
 	Use:                "restore-quorum",
-	Short:              "When quorum is lost, restore quorum to the remaining healthy mon",
+	Short:              "When quorum is lost, restore quorum to the remaining healthy mon.Optional arg 'mon-name'",
```

Comment:
I'd suggest keep the message simple, no need to say it's an optional arg.
```suggestion
	Short:              "When quorum is lost, restore quorum to the remaining healthy mon.",
```

Reply:
not valid anymore.